miniapp/src/test/java/com/rakuten/tech/mobile/miniapp/api/ManifestApiSpec.kt

```diff
@@ -34,42 +31,40 @@ open class ManifestApiSpec private constructor(
 
     internal fun createResponse(
         files: List<String> = listOf(TEST_URL_HTTPS_1, TEST_URL_HTTPS_2)
-    ): MockResponse {
-        val manifestInfo = hashMapOf(
-            "files" to files
+    ): MockResponse = MockResponse().setBody(
+        Gson().toJson(
+            hashMapOf(
+                "files" to files
+            )
         )
-
-        return MockResponse().setBody(Gson().toJson(manifestInfo))
-    }
+    )
 }
 
 class ManifestApiRequestSpec : ManifestApiSpec() {
 
     @Test
     fun `should fetch files information of a mini app using the 'manifest' endpoint`() {
-        mockServer.enqueue(createResponse())
-
-        retrofit.create(ManifestApi::class.java)
-            .fetchFileListFromManifest(
-                miniAppId = TEST_ID_MINIAPP,
-                versionId = TEST_ID_MINIAPP_VERSION
-            ).execute()
-
-        mockServer.takeRequest().requestUrl!!.encodedPath shouldEndWith "manifest"
+        executeManifestCallByRetrofit()
+        val requestUrl = mockServer.takeRequest().requestUrl!!
+        requestUrl.encodedPath shouldEndWith "manifest"
+        requestUrl.encodedQuery.toString() shouldContain "hostVersion=$TEST_HA_ID_VERSION"
```

Comment:
You may want to separate this into two individual test cases.

Reply:
I thought to keep them together because both are existing for the manifest endpoint and if one is not present then technically the test should report failure.